package.json

```diff
@@ -15,7 +15,6 @@
     "preintegration": "npm pack",
     "integration": "cross-env LC_ALL=en-US jest --config test/config/integration.json",
     "lint": "eslint . --ignore-path .gitignore",
-    "prepublish": "npm run build",
```

Comment:
Revert this, please. You're literally breaking the entire package with this deletion.